src/umpire/util/Platform.hpp

```diff
@@ -10,6 +10,7 @@
 namespace umpire {
 
 enum class Platform {
+  none,
```

Comment:
Added new "none" platform for the NullResource.

Reply:
I think is should be "zero" or "null" instead of "none".

Reply:
I think this could/should be "any" since Platform is the corresponding execution place for a memory resource.

Reply:
Now that I think about it, I think the perfect platform for this to come from is "void".